tests/examples/text/awful/keygrabber/autostart.lua

```diff
@@ -0,0 +1,26 @@
+
+local awful = { keygrabber = require("awful.keygrabber") } --DOC_HIDE
+
+local naughty = { notify = function() end } --DOC_HIDE
+
+local autostart_works = false --DOC_HIDE
+
+awful.keygrabber {
+    autostart      = true,
+    release_key    = "Return",
```

Comment:
Hm... perhaps `release_key` should be renamed to `stop_key`? I feel like that could make the function more intuitive.

Reply:
done